taskcluster/ci/raptor/kind.yml

```diff
@@ -8,7 +8,7 @@ transforms:
 kind-dependencies:
     - signing
 only-for-build-types:
-    - raptor
+    - performance-test
```

Comment:
Because of the way indexes are created, I cannot name this build type `raptor` anymore. It's named `performance-test` which I think is a good thing overall. We started to use this name across the board.

Reply:
Naming sounds good to me, it's more generic and useful for upcoming other performance testing we might add in the future 👍